src/test/java/org/broadinstitute/hellbender/tools/exome/CreateAllelicPanelOfNormalsIntegrationTest.java

```diff
@@ -0,0 +1,119 @@
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.pon.allelic.AllelicPanelOfNormals;
+import org.broadinstitute.hellbender.tools.pon.allelic.AllelicPoNTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Stream;
+
+import static org.broadinstitute.hellbender.tools.exome.CreateAllelicPanelOfNormals.*;
+
+/**
+ * Integration tests for {@link CreateAllelicPanelOfNormals}.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+public class CreateAllelicPanelOfNormalsIntegrationTest extends CommandLineProgramTest {
+    private static final String TEST_SUB_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/exome/";
+
+    private static final FileFilter PULLDOWN_FILTER = new WildcardFileFilter("allelic-pon-test-pulldown-*tsv");
+    private static final File[] PULLDOWN_FILES = new File(TEST_SUB_DIR).listFiles(PULLDOWN_FILTER);
+    private static final String[] PULLDOWN_FILE_ARGUMENTS = Stream.of(PULLDOWN_FILES)
+            .map(f -> Arrays.asList("--" + StandardArgumentDefinitions.INPUT_LONG_NAME, f.getAbsolutePath()))
+            .flatMap(Collection::stream)
+            .toArray(size -> new String[2 * PULLDOWN_FILES.length]);
+
+    private static final File PON_EXPECTED_SITE_FREQUENCY_50_TSV = new File(TEST_SUB_DIR, "allelic-pon-test-pon-freq-50.tsv");
+    private static final File PON_EXPECTED_SITE_FREQUENCY_75_TSV = new File(TEST_SUB_DIR, "allelic-pon-test-pon-freq-75.tsv");
+
+    private static final File PON_EXPECTED_SITE_FREQUENCY_50_HDF5 = new File(TEST_SUB_DIR, "allelic-pon-test-pon-freq-50.pon");
+    private static final File PON_EXPECTED_SITE_FREQUENCY_75_HDF5 = new File(TEST_SUB_DIR, "allelic-pon-test-pon-freq-75.pon");
+
+    private static final String TEMP_HDF5_OUTPUT_FILE_NAME = createTempFile("create-allelic-pon-test", ".pon").getAbsolutePath();
+    private static final String TEMP_TSV_OUTPUT_FILE_NAME = createTempFile("create-allelic-pon-test", ".tsv").getAbsolutePath();
+
+    @BeforeTest
+    void deleteTempFilesToBeCreated() throws IOException {
+        Files.delete(Paths.get(TEMP_HDF5_OUTPUT_FILE_NAME));
+        Files.delete(Paths.get(TEMP_TSV_OUTPUT_FILE_NAME));
+    }
+
+    @DataProvider(name = "dataArgumentValidation")
+    public Object[][] dataArgumentValidation() {
+        return new Object[][]{
+                {new String[]{"--" + SITE_FREQUENCY_LONG_NAME, "0"}},
+                {new String[]{"--" + SITE_FREQUENCY_LONG_NAME, "10"}},
+                {new String[]{"--" + SITE_FREQUENCY_LONG_NAME, "-1"}}
```

Comment:
This is another one that we are inconsistent on (actually it looks like we seem prefer it without the space).  Perhaps a good guideline would be to keep things consistent within each class, at least.